man/systemd-cryptenroll.xml

```diff
@@ -545,6 +547,14 @@
     <para>On success, 0 is returned, a non-zero failure code otherwise.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Examples</title>
+
+    <para><citerefentry><refentrytitle>crypttab</refentrytitle><manvolnum>5</manvolnum></citerefentry> and
+    <citerefentry><refentrytitle>systemd-measure</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+    contains various examples employing <command>systemd-cryptenroll</command>.</para>
+  </refsect1>
```

Comment:
Yeah, that's a possibility. I can change it to xinclude if people like that better… I wanted to avoid confusing the reader and artificially increasing the amount of text by having the same lengthy example in multiple places.